app/views/mass_rollouts/review_deploy.html.erb

```diff
@@ -28,6 +28,16 @@
         </table>
       </div>
 
+      <div class="col-lg-4">
+        <label>Deploy reference from:</label>
+        <%= select_tag :reference_source, options_for_select([
```

Comment:
apply some bootstrap classes to make it look nice?

Reply:
I'd also position this right aligned closer to the actual button, but either/or.